public/opensource-projects/index.json

```diff
@@ -0,0 +1,255 @@
+    "lang": ["C++", "Python"],
+    "issue": "https://github.com/elementsproject/elements/issues"
+  },
+  "pyblock": {
+    "repo_url": "https://github.com/curly60e/pyblock",
+    "name": "pyblock",
+    "org": "",
+    "tags": ["bitcoin"],
+    "lang": ["C++", "Python"],
+    "issue": "https://github.com/curly60e/pyblock/issues"
+  },
+  "bitcoinj": {
+    "repo_url": "https://github.com/bitcoinj/bitcoinj",
+    "name": "bitcoinj",
+    "org": "bitcoinj",
+    "tags": ["bitcoin", "library", "blockchain", "bitcoinj", "taproot"],
+    "lang": ["Java"],
+    "issue": "https://github.com/bitcoinj/bitcoinj/issues"
+  },
+  "bitbox-wallet-app": {
+    "repo_url": "https://github.com/digitalbitbox/bitbox-wallet-app",
+    "name": "bitbox-wallet-app",
+    "org": "digital-bit-box",
+    "tags": ["bitcoin", "hardware-wallet", "wallet", "segwit"],
+    "lang": ["Go", "Typescript"],
+    "issue": "https://github.com/digitalbitbox/bitbox-wallet-app/issues"
+  },
+  "trezor-firmware": {
+    "repo_url": "https://github.com/trezor/trezor-firmware",
+    "name": "trezor-firmware",
+    "org": "Trezor",
+    "tags": ["cryptography", "bitcoin", "firmware", "cryptocurrency", "wallet"],
+    "lang": ["C", "Python"],
+    "issue": "https://github.com/trezor/trezor-firmware/issues"
+  },
+  "joinmarket-clientserver": {
+    "repo_url": "https://github.com/JoinMarket-Org/joinmarket-clientserver",
+    "name": "joinmarket-clientserver",
+    "org": "JoinMarket-Org",
+    "tags": ["privacy", "bitcoin", "coinjoin", "joinmarket"],
+    "lang": ["Python"],
+    "issue": "https://github.com/JoinMarket-Org/joinmarket-clientserver/issues"
+  },
+  "lightning": {
+    "repo_url": "https://github.com/ElementsProject/lightning",
+    "name": "lightning",
+    "org": "ElementsProject",
+    "tags": ["lightning", "bitcoin", "p2p", "lightning-network"],
+    "lang": ["C", "Python"],
+    "issue": "https://github.com/ElementsProject/lightning/issues"
+  },
+  "pool": {
+    "repo_url": "https://github.com/lightninglabs/pool",
+    "name": "pool",
+    "org": "Lightning labs",
+    "tags": ["lightning", "lightning-network"],
+    "lang": ["Go"],
+    "issue": "https://github.com/lightninglabs/pool/issues"
+  },
+  "warden_terminal": {
+    "repo_url": "https://github.com/pxsocs/warden_terminal",
+    "name": "warden_terminal",
+    "org": "",
+    "tags": ["bitcoin", "mempool"],
+    "lang": ["Javascript", "Python"],
+    "issue": "https://github.com/pxsocs/warden_terminal/issues"
+  },
+  "robosats": {
+    "repo_url": "https://github.com/RoboSats/robosats",
+    "name": "robosats",
+    "org": "RoboSats",
+    "tags": ["bitcoin", "privacy", "lightning", "p2p", "tor", "exchange", "i2p"],
+    "lang": ["Typescript", "Python"],
+    "issue": "https://github.com/RoboSats/robosats/issues"
+  },
+  "thunderhub": {
+    "repo_url": "https://github.com/apotdevin/thunderhub",
+    "name": "thunderhub",
+    "org": "",
+    "tags": ["lightning", "lightning-network"],
```

Comment:
Why I decided to add`lightning-network` is some repositories have both lightning and lightning-network in their tags

Reply:
Don't they mean the same thing? I mean we can have just one and whenever a user searches for lightning, they'd still see all projects related to lightning regardless of whether there's lightning network or not because lightning is a keyword in "lightning-network".